shared/common-adapters/relative-popup-hoc.desktop.tsx

```diff
@@ -347,13 +347,17 @@ function ModalPositionRelative<PP>(
     )
 
     componentDidMount() {
-      document.body && document.body.addEventListener('click', this._handleClick)
-      document.body && document.body.addEventListener('scroll', this._handleScroll, true)
+      const node = document.body
+      if (!__STORYBOOK__ && node) {
```

Comment:
don't do this at all in stories, messes up clicks in popups
also the scroll thing did nothing